packages/ResizeObserver/src/ResizeObserver.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useResizeObserver from "use-resize-observer/polyfilled";
+import debounce from "lodash.debounce";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./ResizeObserver.styles";
+
+const propTypes = {
+  breakpointLarge: PropTypes.number,
+  breakpointSmall: PropTypes.number,
+  children: PropTypes.node,
+  debounceDelay: PropTypes.number,
+  isFullWidth: PropTypes.bool,
+  isFullHeight: PropTypes.bool,
+  onBreak: PropTypes.func,
+  onResize: PropTypes.func,
+};
+
+const defaultProps = {
+  breakpointLarge: 768,
+  breakpointSmall: 360,
+  children: null,
+  debounceDelay: 200,
```

Comment:
Not sure about a default value for this?

Reply:
I feel like we could go lower by default.

Reply:
`30` seems barely perceptible.  We could go with that be default and if there's too much of a performance hit with heavy components, consumers would be responsible to increase the delay. Sound more reasonable?